src/main/java/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.java

```diff
@@ -205,16 +205,33 @@ private Optional<HmppsDomainEvent> toPrisonerReleased(OffenderEvent event) {
     }
 
     private Optional<HmppsDomainEvent> toCaseNotePublished(OffenderEvent event) {
-        return Optional.of(HmppsDomainEvent.builder()
-            .eventType("prison.case-note.published")
-            .description("A prison case note has been created or amended")
-            .detailUrl(String.format("%s/case-notes/%s/%d", offenderEventsProperties.getCasenotesApiBaseUrl(), event.getOffenderIdDisplay(), event.getCaseNoteId()))
-            .occurredAt(toOccurredAt(event))
-            .publishedAt(OffsetDateTime.now().toString())
-            .personReference(new PersonReference(event.getOffenderIdDisplay()))
-            .build()
-            .withAdditionalInformation("caseNoteId", event.getCaseNoteId().toString())
-            .withAdditionalInformation("caseNoteType", event.getEventType()));
+        if (event.getCaseNoteSubType() == null) {
```

Comment:
what is the significance of this being null?

Reply:
Ah yes - this happens to identify the old / existing case note event which has the case note type/subtype packed into the 'eventType' field.